tools/xform-baseline-1.sh

```diff
@@ -62,6 +62,14 @@ sed -i "s/test\/gst-vaapi\/encode\/vp9.py:test_8bit_cqp(/test\/gst-vaapi\/encode
 sed -i "s/test\/gst-vaapi\/encode\/vp9.py:test_8bit_cbr(/test\/gst-vaapi\/encode\/vp9.py:cbr.test(/g" $1
 sed -i "s/test\/gst-vaapi\/encode\/vp9.py:test_8bit_vbr(/test\/gst-vaapi\/encode\/vp9.py:vbr.test(/g" $1
 
+sed -i "s/test\/gst-vaapi\/vpp\/brightness.py:test_default(/test\/gst-vaapi\/vpp\/brightness.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/contrast.py:test_default(/test\/gst-vaapi\/vpp\/contrast.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/hue.py:test_default(/test\/gst-vaapi\/vpp\/hue.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/saturation.py:test_default(/test\/gst-vaapi\/vpp\/saturation.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/denoise.py:test_default(/test\/gst-vaapi\/vpp\/denoise.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/mirroring.py:test_default(/test\/gst-vaapi\/vpp\/mirroring.py:default.test(/g" $1
+sed -i "s/test\/gst-vaapi\/vpp\/sharpen.py:test_default(/test\/gst-vaapi\/vpp\/sharpen.py:default.test(/g" $1
+
```

Comment:
missing deinterlace and csc transforms

Reply:
It should be no influence on the two transforms in this patch

Reply:
Yes, you are correct... sorry.